src/main/resources/application.yaml

```diff
@@ -16,28 +16,36 @@ spring:
   config:
     import: "optional:configtree:/mnt/secrets/"
   application:
-    name: Spring Boot Template
-#  datasource:
-#    driver-class-name: org.postgresql.Driver
-#    url: jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}${DB_OPTIONS:}
-#    username: ${DB_USER_NAME}
-#    password: ${DB_PASSWORD}
-#    properties:
-#      charSet: UTF-8
+    name: PIP Subscription Management
+  datasource:
+    driver-class-name: org.postgresql.Driver
+    url: jdbc:postgresql://${db_name}.postgres.database.azure.com:${db_port}/postgres
+
+    username: ${db_user}
+    password: ${db_pass}
+    properties:
+      charSet: UTF-8
 #    hikari:
 #      minimumIdle: 2
 #      maximumPoolSize: 10
 #      idleTimeout: 10000
 #      poolName: {to-be-defined}HikariCP
 #      maxLifetime: 7200000
 #      connectionTimeout: 30000
-#  jpa:
-#    properties:
-#      hibernate:
-#        jdbc:
-#          lob:
-#            # silence the 'wall-of-text' - unnecessary exception throw about blob types
-#            non_contextual_creation: true
+  jpa:
+    show-sql: 'true'
```

Comment:
I think the show-sql should be removed as it currently populates the logs with details of each query made. Happy to discuss though!